### ISSUE TITLE
Fonts don't work when you install the package

### ISSUE THREAD

**Issue by dlcomeaux** [CMAP-REPOS/cmapplot#71]
I tried installing the package to use it on a graph I'm making, and it appears that the font issues we thought we'd solved are **not** solved. When I attempt to create a graph using `theme_cmap` it gives the following (and annoyingly familiar) error
message:

```
Warning messages:
1: In grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  :
  font family not found in Windows font database
2: In grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  :
  font family not found in Windows font database
3: In grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  :
  font family not found in Windows font database
4: In grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  :
  font family not found in Windows font database
```

As a result, it is exporting text in Arial, and not Calibri. The issue persists even after running `finalize`.

I'm not really sure how best to proceed on fixing this, as I really thought we had. But flagging for others to test if they get the same issue. In either case, we need to do some investigating.

**Comment by dlcomeaux** [comment]
@nmpeterson and @matthewstern in particular, curious if you have thoughts here.

**Comment by matthewstern** [comment]
As we consider this, I'd like to remind us of the change implemented [here](https://github.com/CMAP-REPOS/cmapplot/pull/53#issuecomment-697065629). There's a chance @nmpeterson's original implementation of fonts did work, but this change (to get PDF exporting to work) created this error that went undiscovered until now.

**Comment by dlcomeaux** [comment]
Update: if you manually run `windowsFonts('Calibri' = windowsFont('Calibri'), 'Calibri Light' = windowsFont('Calibri Light'))`, it works. But, that code is already in the package in `cmapplot.R`, so I'm not sure why it isn't running on install. That seems like a question about how installation works, which I don't have a good grasp of.

**Comment by nmpeterson** [comment]
I think this gets to the underlying reason: https://r-pkgs.org/r.html#understand-when-code-is-executed

**Comment by dlcomeaux** [comment]
So would doing something like [this](https://stackoverflow.com/questions/20223601/r-how-to-run-some-code-on-load-of-package) help?

**Comment by nmpeterson** [comment]
Looks promising!

**Comment by matthewstern** [comment]
Addressed as of #72
